packages/next-swc/crates/core/src/relay.rs

```diff
@@ -0,0 +1,276 @@
+
+                        match group {
+                            Ok(group) => match group {
+                                FileGroup::Source { source_set } => match source_set {
+                                    SourceSet::SourceSetName(source_set_name) => {
+                                        let project_config: Option<&ProjectConfig> =
+                                            config.projects.get(&source_set_name);
+
+                                        match project_config {
+                                            None => Err(BuildRequirePathError::ProjectNotFoundForSourceSet { source_set_name }),
+                                            Some(project_config) => {
+                                                Ok(path_for_artifact(&config.root_dir,real_file_name, &project_config, operation_name))
+                                            }
+                                        }
+                                    }
+                                    SourceSet::SourceSetNames(source_set_names) => {
+                                        Err(BuildRequirePathError::MultipleSourceSetsFound {
+                                            source_set_names,
+                                            path: real_file_name,
+                                        })
+                                    }
+                                },
+                                _ => Err(BuildRequirePathError::FileNotASourceFile),
+                            },
+                            Err(err) => Err(BuildRequirePathError::CouldNotCategorize {
+                                err,
+                                path: real_file_name.display().to_string(),
+                            }),
+                        }
+                    }
+                    _ => Err(BuildRequirePathError::FileNameNotReal),
+                }
+            }
+        }
+    }
+
+    fn build_call_expr_from_tpl(&mut self, tpl: &TaggedTpl) -> Option<Expr> {
+        if let Expr::Ident(ident) = &*tpl.tag {
+            if &*ident.sym != "graphql" {
+                return None;
+            }
+        }
+
+        let operation_name = pull_first_operation_name_from_tpl(tpl);
+
+        match operation_name {
+            None => None,
+            Some(operation_name) => match self.build_require_path(operation_name.as_str()) {
+                Ok(final_path) => Some(build_require_expr_from_path(final_path.to_str().unwrap())),
+                Err(err) => {
+                    let base_error = "Could not transform GraphQL template to a Relay import.";
+                    let error_message = match err {
+                        BuildRequirePathError::FileNameNotReal => "Source file was not a real \
+                                                                   file. This is likely a bug and \
+                                                                   should be reported to Next.JS"
```

Comment:
Small change in the casing:
```suggestion
                                                                   should be reported to Next.js"
```